docs/test-specifications/contract-service/ContractCreateTransaction.md

```diff
@@ -0,0 +1,323 @@
+| 1      | Create contract with admin key and reasonable gas             | adminKey=valid, gas="1000000"          | Transaction succeeds, contract deployed. | N           |
+| 2      | Create contract without admin key and reasonable gas          | adminKey=empty, gas="1000000"          | Transaction succeeds, contract deployed. | N           |
+| 3      | Create contract with admin key and zero gas                   | adminKey=valid, gas="0"                | Fails with `INSUFFICIENT_GAS`.           | N           |
+| 4      | Create contract without admin key and zero gas                | adminKey=empty, gas="0"                | Fails with `INSUFFICIENT_GAS`.           | N           |
+| 5      | Create contract with admin key and excessive gas over limit   | adminKey=valid, gas="9223372036854775807" | Fails with `MAX_GAS_LIMIT_EXCEEDED`.     | N           |
+| 6      | Create contract without admin key and excessive gas over limit| adminKey=empty, gas="9223372036854775807" | Fails with `MAX_GAS_LIMIT_EXCEEDED`.     | N           |
+| 7      | Create contract with admin key and negative gas               | adminKey=valid, gas="-1"               | Fails with `CONTRACT_NEGATIVE_GAS`.      | N           |
+| 8      | Create contract without admin key and negative gas            | adminKey=empty, gas="-1"               | Fails with `CONTRACT_NEGATIVE_GAS`.      | N           |
+| 9      | Create contract without admin key and gas 9,223,372,036,854,775,806 (int64 max - 1)  |adminKey=empty, gas="9223372036854775806"| The transaction succeeds, contract deployed | N  |
+| 10     | Create contract with admin key and gas 9,223,372,036,854,775,806 (int64 max - 1)  |adminKey=valid, gas="9223372036854775806"| The transaction succeeds, contract deployed | N   |
+| 11     | Create contract with admin key and gas 9,223,372,036,854,775,807 (int64 max)  |adminKey=valid, gas="9223372036854775807"| The transaction succeeds, contract deployed | N   |
+| 12     | Create contract without admin key and gas 9,223,372,036,854,775,807 (int64 max)  |adminKey=empty, gas="9223372036854775807"| The transaction succeeds, contract deployed | N   |
+| 13     | Create contract with admin key and gas -9,223,372,036,854,775,808 (int64 min)  |adminKey=valid, gas="-9,223,372,036,854,775,808"| The transaction fails | N   |
+| 14     | Create contract without admin key and gas -9,223,372,036,854,775,808 (int64 min)  |adminKey=empty, gas="-9,223,372,036,854,775,808"| The transaction fails| N |
+| 15     | Create contract with admin key and gas -9,223,372,036,854,775,807 (int64 min + 1)  |adminKey=valid, gas="-9,223,372,036,854,775,807"| The transaction fails| N   |
+| 16     | Create contract without admin key and gas -9,223,372,036,854,775,807 (int64 min + 1)  |adminKey=empty, gas="-9,223,372,036,854,775,807"| The transaction fails | N |
```

Comment:
The tests with `adminKey=empty` can just not provide an `adminKey` input. Also `adminKey=valid` --> `adminKey=<VALID_ADMIN_KEY>` to stay consistent

Reply:
This is the same for `Constructor Parameters` tests and `Max Automatic Associations`.